circle.yml

```diff
@@ -15,12 +15,14 @@ checkout:
 
 dependencies:
   pre:
-    - sudo apt-get update && sudo apt-get install valgrind
+    - sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-5.0 main'
+    - sudo apt-get update && sudo apt-get install valgrind clang-format-5.0
 
 test:
   pre:
     - ./configure -i
   override:
+    - ./tools/clang-format-check.sh
```

Comment:
Does this get executed before the build now?

Reply:
Yes